2018/10/16/pom.xml

```diff
@@ -29,6 +33,11 @@
           </execution>
         </executions>
         <configuration>
+          <archive>
+            <manifest>
+              <mainClass>org.openjdk.jmh.Main</mainClass>
```

Comment:
IMO this is the preferred way to set up the "main" for a JMH project. You then don't need any main class, and you run the thing simply like: `java -jar path/to/jar/microbencmarks.jar`, and the main inside parses all the arguments. JMH has many useful arguments, so this lets you actually use them. In particular, I use `-prof perfnorm` and `-prof perfasm` to the the JMH-perf integration to view performance counters, and annotated assembly.